src/element/util.ts

```diff
@@ -28,7 +28,7 @@ export function deepMerge<T>(obj1: T, obj2: T): T {
         if (!Object.hasOwn(obj2 as object, k)) continue
         if (obj1[k] instanceof Object && obj2[k] instanceof Object) {
             res[k] = deepMerge(obj1[k], obj2[k])
-        } else {
+        } else if (obj2[k] != null) {
```

Comment:
We took null into consideration because if the value is saved as null for some reason, it cannot be restored.